opennms-config-tester/src/main/java/org/opennms/netmgt/config/tester/ConfigTester.java

```diff
@@ -90,6 +98,33 @@ public void testConfig(String name, boolean ignoreUnknown) {
 		}
 	}
 
+	private void checkDirectoryForXmlFiles(final String name, final String xmlModelName) {
+		final Class clazz;
+		try {
+			clazz = Class.forName(xmlModelName);
+		} catch (ClassNotFoundException e) {
+			throw new RuntimeException(e);
+		}
+
+		final Path directory = Paths.get(ConfigFileConstants.getFilePathString(), name);
+
+		if (!Files.exists(directory)) {
+			return;
+		}
```

Comment:
Can we check first for the directory and afterwards try to load the class? 
There is no need to load the class, if nothing needs to be done.

Reply:
done